### PR TITLE
Added ability to delete users

### PR DESCRIPTION
**Pull request by danieljtate**

### CODE REVIEW

File: app/components/Users.js [hunk 1]
```diff
@@ -428,13 +485,13 @@ gatherAbsences: function() {
       self.setState({
         approvedAbsences: absenceData.data
       });
+      console.log(approvedAbsences);
```
Comment (AlexJYoungman) [inline app/components/Users.js]: Remove Console Logs

File: app/components/Users.js [hunk 2]
```diff
@@ -369,37 +367,96 @@ editUser: function () {
...
+    }
+    axios.all(promises)
+    .then(response => {
+      axios.delete(ServiceURL + 'user/' + this.state.userInfo.userId, { headers: {  'Authorization': 'Bearer ' + window.localStorage.getItem('token') } })
+      .then(response => {
+        this.refreshPage();
+      })
+    })
+    }
+  else {
```
Comment (AlexJYoungman) [inline app/components/Users.js]: Sort indentation here

File: app/components/Users.js [hunk 3]
```diff
@@ -473,27 +530,45 @@ absenceApprovers: function () {
...
+absenceApproversDelete: function () {
+  var self = this;
+  var approversAbsences = [];
+  var matchedAbsences = [];
+  var editedApprover = this.state.userInfo.userId;
+  for (var i = 0; i < this.state.approvedAbsences.length; i++) {
+  if (this.state.approvedAbsences[i].approverId.userId === editedApprover){
+    matchedAbsences.push(this.state.approvedAbsences[i]);
+  }
+}
```
Comment (AlexJYoungman) [inline app/components/Users.js]: Indentation

File: app/components/Users.js [hunk 4]
```diff
@@ -331,13 +329,13 @@ var Users = React.createClass({
...
-userInfo: function (obj) {
-
-this.setState({
-  userInfo: obj
-})
-this.gatherAbsences();
+  this.setState({
+    userInfo: obj
+  })
+  console.log("hit");
```
Comment (AlexJYoungman) [inline app/components/Users.js]: Remove Console Logs

File: app/components/Users.js [hunk 5]
```diff
@@ -369,37 +367,93 @@ editUser: function () {
...
+      .then(response => {
+        this.refreshPage();
+      })
+    })
+  }
+  else {
+    axios.delete(ServiceURL + 'user/' + this.state.userInfo.userId, { headers: {  'Authorization': 'Bearer ' + window.localStorage.getItem('token') } })
+    .then(response => {
+    })
+    this.refreshPage();
```
Comment (AlexJYoungman) [inline app/components/Users.js]: The refresh should be inside the `.then` ?

File: app/styles/css/users.css [hunk 6]
```diff
@@ -597,6 +597,15 @@ input[type=checkbox]:checked +label {
   margin-left: 35%;
 }
 
+#deleteConfirm {
+  height:200px;
+  background-color: white;
+  border-radius: 5px;
+  width:400px;
+  margin-top: 350px;
+  margin-left: 35%;
```
Comment (Ian-Accenture) [inline app/styles/css/users.css]: would `margin: auto; margin-top: 350px;` not be more appropriate here?

File: app/styles/css/users.css [hunk 7]
```diff
@@ -606,6 +615,15 @@ input[type=checkbox]:checked +label {
   margin-left: 35%;
 }
 
+#newApproversDelete {
+  height:300px;
+  background-color: white;
+  border-radius: 5px;
+  width:500px;
+  margin-top: 350px;
+  margin-left: 35%;
```
Comment (Ian-Accenture) [inline app/styles/css/users.css]: would `margin: auto; margin-top: 350px;` not be more appropriate here?

**Pull request merged**